src/jsonrpc.js

```diff
@@ -1,30 +1,104 @@
+import bluebird from 'bluebird';
 import logger from './logger';
+import { isInArray } from './utils';
 
-export class JsonRpc {
+export default class JsonRpc {
   /**
    * Initialize class local variables
-   * @param {*} addresses 
-   * @param {*} fromBlock 
-   * @param {*} toBlock 
-   * @param {*} callback 
+   * @param Array addresses 
+   * @param int currentBlock 
+   * @param int toBlock 
+   * @param function callback 
    */
-  constructor(addresses, fromBlock, toBlock, web3, callback = null) {
+  constructor(addresses, currentBlock, toBlock, web3, callback = null) {
```

Comment:
I recommend this gets changed to `fromBlock` so that it becomes 
`this.currentBlock = fromBlock`